metrics/error-report/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-prototype-builtins */
+import AES from "crypto-js/aes";
+import Utf8 from "crypto-js/enc-utf8";
+import ScriptLoader from "../../integrations/ScriptLoader";
+// import RudderApp from "../../utils/RudderApp";
+
+const metaData = {
+  SDK: {
+    name: "RudderStack JavaScript SDK",
+    version: "process.package_version",
+    installType: process.browser ? "CDN" : "NPM",
+  },
+};
+
+const credentials = {
+  apiKey:
+    "U2FsdGVkX182Myeej4DKayK1qKvTaxfa/9g4oz/kMD1hkuPCBKmyrWZB74x3N0qf6xuPtH8QRq4YRnIh15AaNg==", // "0d96a60df267f4a13f808bbaa54e535c"
+  appVersion: "1.0.0",
```

Comment:
Why does this default to `1.0.0`?

Reply:
what do you suggest?

Reply:
Unless it's a mandatory field, we don't need to send anything as we are not exactly tracking the app here. However, I'd send the SDK version here to differentiate the errors across multiple SDK versions.

Reply:
SDK version is already added.

Reply:
can we filter the errors based on that more easily than this?

Reply:
I am not sure. In plain sight it’s not duable and using a custom filter it can be done but not able to check as we need an upgraded plan

Reply:
Any update here?

Reply:
As discussed, let's use the SDK version for app version here.